src/app/pages/receive-tokens/receive-btc.tsx

```diff
@@ -13,18 +18,41 @@ export function ReceiveBtcModal() {
   const btcAddress = useBtcNativeSegwitAccountIndexAddressIndexZero(accountIndex);
   const analytics = useAnalytics();
   const { onCopy } = useClipboard(btcAddress);
+  const navigate = useNavigate();
 
   function copyToClipboard() {
     void analytics.track('copy_btc_address_to_clipboard');
     toast.success('Copied to clipboard!');
     onCopy();
   }
 
+  const ReceiveBtcModalWarning = (
+    <WarningLabel mt="base-loose" width="100%">
+      <Stack>
+        <Text color="#242629" fontSize={1} fontWeight={500}>
+          Do not deposit Ordinals here
+        </Text>
+        <Text>
+          Use the{' '}
+          <Link
+            onClick={() => navigate(RouteUrls.ReceiveCollectible)}
+            color="blue"
+            display="inline-block"
+            textDecoration="underline"
+          >
+            address created for collectibles.
+          </Link>
+        </Text>
+      </Stack>
+    </WarningLabel>
+  );
```

Comment:
This could be a separate component, rather than declared within the receive-btc modal